src/containers/Tenant/Acl/Acl.scss

```diff
@@ -2,21 +2,16 @@
 
 .ydb-acl {
     width: 100%;
-    &__owner-container {
-        padding-bottom: 25px;
-    }
     &__result {
-        padding-bottom: 16px;
-    }
-
-    &__owner {
-        font-weight: 600;
+        padding-bottom: var(--g-spacing-4);
+        padding-left: var(--g-spacing-2);
```

Comment:
Why do we need this padding?

![Screenshot 2024-07-22 at 12 34 25](https://github.com/user-attachments/assets/6d1e79a1-0ccb-40e6-b411-fda3e8f5bf61)

Reply:
Discussed with @adameat that it looks prettier with small padding.